src/types/venues.ts

```diff
@@ -319,6 +319,7 @@ export interface VenueEvent {
   collective_price: number;
   host: string;
   room?: string;
+  venueId?: string;
```

Comment:
Is this going to be on every event always? We have a WithVenueId helper.

Reply:
we are going to prepopulate our data with it. So in this environment, yes

Reply:
Makes sense.